### DISCUSSION TITLE
Higher Circle size value gives you smaller circles

### DISCUSSION THREAD

**Discussion by LOKSTED** [ppy/osu#13182] [Ideas]
**Describe the bug:** 
In the difficulty adjust mod, the slider for circle size should be the other way around?
**Screenshots or videos showing encountered issue:** 
![osu_2021-02-25_02-01-59](https://user-images.githubusercontent.com/29303173/109122135-c3e86680-770d-11eb-8a94-85ba417444ac.jpg)

![osu_2021-02-25_02-02-30](https://user-images.githubusercontent.com/29303173/109122062-ac10e280-770d-11eb-8b64-4e1ce40a8454.jpg)

**osu!lazer version:** 
2021.220.0

**Comment by frenzibyte** [comment]
That's how it's usually is in stable, the greater difficulty values mean the more beatmap is difficult.

Though I kinda agree the name is misleading to what it actually does, maybe rename it to `Circle Smallness` or similar? or maybe add a note that greater CS means smaller circles.

**Comment by bdach** [comment]
I would rather flip the values or describe it better via other means than name it "smallness".

**Comment by Contextualist** [comment]
> I would rather flip the values

I think that keeping a consistent story of "higher values stand for greater difficulty" is important. e.g. maintaining a uniform visual hint for the difficulty bars.

I like the name `Circle Smallness` as it does not change the abbreviation CS, which is generally accepted by the community.

**Comment by yellowsink** [comment]
I like the idea of renaming it, but "Circle Smallness" sounds really weird imo

**Comment by ambeeeeee** [comment]
Since OD was renamed to `Accuracy` it could just be renamed to `Aim` or `Aim Precision` or `Precision`? I feel like `Aim Precision` would be more clear but feels kinda verbose. `Aim` would fall in line with `Accuracy` but it could be unclear how it affects the map. If that's a concern they could have hover text on all of them to give a brief explanation of how they affect the map.

**Comment by peppy** [comment]
All the terms you've suggested change the display from being one which explains its purpose to one which requires a tooltip or description to know exactly what it does.

**Comment by yyny** [comment]
"Circle Hitbox Difficulty"

**Comment by frenzibyte** [comment]
Sounds misleading for one that basically shrinks the size of the hit object, which would lead to [the above](https://github.com/ppy/osu/issues/11899#issuecomment-816984176).

**Comment by Icymice** [comment]
I'd either suggest keeping it the same, as most players know that's how CS works at this point. If we had to rename it I'd suggest "Circle Shrink" as it's simple enough for most players and keeps the 'CS' initials.

**Comment by ShiiTsuin** [comment]
> I'd either suggest keeping it the same, as most players know that's how CS works at this point. If we had to rename it I'd suggest "Circle Shrink" as it's simple enough for most players and keeps the 'CS' initials.

"Circle Shrink" doesn't really make sense when you consider that CS is a value set by mappers to determine the **size** of the **circles**. "Circle Shrink" sounds like an alternative name to the Deflate mod to me hahahaha

Ideally, CS values should be flipped so that a larger circle size setting correlates to a larger hit circle, though that obviously contradicts the idea of "high setting = harder" seen with the other difficulty settings
